packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -914,17 +920,25 @@ export class MgtPersonCard extends MgtTemplatedComponent {
    * @memberof MgtPersonCard
    */
   protected renderMessagingSection(): TemplateResult {
-    return html`
-         <fluent-text-field appearance="filled" placeholder="Message ${this.internalPersonDetails.displayName}"
-           .value=${this._chatInput}
-           @input=${(e: Event) => {
-             this._chatInput = (e.target as HTMLInputElement).value;
-           }}>
-         </fluent-text-field>
-         <span class="send-message-icon" @click=${() => this.sendQuickMessage()}>
-           ${getSvg(SvgIcon.Send)}
-         </span>
-       `;
+    const person = this.personDetails as User;
```

Comment:
Is there an opportunity here to refactor this quick message section into a standalone component? 
Some of our components are very big and if we take the opportunity to make more smaller components it lowers the cognitive load when it comes time to make changes in the future

Reply:
All sections should be smaller components. Files, emails, org chart, etc.

Reply:
@gavinbarron @sebastienlevert @Mnickii I would like to note a few things about this [standalone components] and theming.

When you use another mgt component inside mgt, like in the person-card scenario, I had to import the various themes of the _sub-components_ so that their theming can pick (esp for dark theme). Otherwise, you end up spending a lot of time wondering why some CSS changes are not working _yet_ you've done everything correctly. Have a look at this line here https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/fa154a33e655ec40e818a864870b74f52e3ee554/packages/mgt-components/src/components/mgt-person-card/mgt-person-card.theme.scss#L10-L19

One advantage I have also encountered with creating themes for specific components is the ease of sharing the CSS tokens. For example, the `mgt-profile.theme` file now exposes an `--token-overflow-color` token. Now, all I had to do was show that the token can be used as part of the tokens to customize `mgt-person-card` without changing any of it's CSS (kind of inherited). IMO I think this is how it should be done especially for a component like `mgt-person-card` which utilizes many mgt components but makes it challenging to customize. This is cos we rename tokens which we can use as they are. For example: https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/fa154a33e655ec40e818a864870b74f52e3ee554/packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts#L71-L79

These are `mgt-person` tokens but that rename makes you redefine them twice(in mgt-person which should be the only place they're defined, and now in mgt-person-contact where the token names are changed).

I want to pursue this further and sanitize most of these tokens during FHL however I'll need help in documentation and QA. I believe many tokens will break for v3. What do you all think?